crates/polars-ops/src/series/ops/cut.rs

```diff
@@ -137,7 +137,13 @@ pub fn qcut(
 
     if ca.null_count() == ca.len() {
         // If we only have nulls we don't have any breakpoints.
-        return cut(&s, vec![], labels, left_closed, include_breaks);
+        return cut(
```

Comment:
Can we directly return a `full_null` with the proper type?

Reply:
I believe so. So if all values are null, there's actually no need to call cut, correct? 
I can make this change

Reply:
Yes, that's the idea.

Reply:
Added it so that this can be included in the release.